userguide/kaui/includes/tenants.adoc

```diff
@@ -1,54 +1,338 @@
++
+image::CatalogConfiguration.png[width=450,align="center"]
++
+. Fill in the fields. For field descriptions, see <<Catalog Configuration Field Descriptions>>.
+. Click the *Save* button.
+
+==== Catalog Configuration Field Descriptions
+
+For additional field information, see the https://killbill.github.io/slate/#catalog[Catalog API].
+
+[cols="1,3"]
+|===
+^|Field ^|Description
+
+| Product Category
+a| Define whether this product is of the `BASE`, `ADDON`, or `STANDALONE` category:
+
+* Base products can have one or more addons.
+
+* Add-on products can be bundled with a base product.
+
+* Standalone products cannot have any add-ons.
+
+| Product Name
+| The name assigned to the product you are selling.
+
+| Plan Name
+| The name of the plan, which defines how the product will be sold (for example, a monthly subscription or a one-time purchase).
+
+| Amount
+| The price of the plan to be paid every billing period.
+
+| Currency
+| The currency this plan uses. If you need to add more currencies, you can do so by <<Add a Currency to a Plan,adding a currency to a plan>>.
+
+| Billing Period
+| The period for which the customer is billed.
+
+| Trial Length
+| Along with a unit of time (*Trial Time Unit* below), defines the length of the trial.
+
+| Trial Time Unit
+| Along with the *Trial Length* number above, specifies the time interval for the trial.
+|===
+
+==== Add a Currency to a Plan
+
+_I'm having trouble with this functionality. I get the error message "Error while creating plan: plan [NAME] already exists." There might be some rules around creating a new currency. Maybe can't do it with a simple plan?_
+
+. <<View Catalog Products and Plans,Go to the Catalog Show tab>> on the Tenant Configuration page.
++
+image::Catalog-Tabs-ShowCat.png[width=850,align="center"]
++
+. On the row of the plan for which you want to add a currency, click *+ currency* on the far right.
++
+Kaui displays the Add Plan Currency screen:
++
+image::Catalog-AddPlanCurrency.png[width=450,align="center"]
+. Fill in the fields:
++
+* *Plan ID*—By default, Kaui fills in this field based on the plan you selected, but you can change it to a different plan ID.
+* *Amount*—The cost of the plan in the new currency you're adding.
+* *Currency*—The currency you are adding.
++
+. Click the *Save* button. Kaui displays the new currency on the Catalog Show tab.
+
+==== Upload an XML Catalog
+
+_Something I'm not sure of is if you create a simple catalog, can you overwrite it by uploading an XML file? Seems like you should be able to do this, but I get an error when I attempt it. First, I get the Cloudflare error if I haven't removed the <?xml version="1.0" encoding="UTF-8" standalone="no"?> on the first line. After I clear that, I still get the KB error: "Error while communicating with the Kill Bill server: Invalid catalog for tenant: [TENANT #]." When I upload the same catalog on a new tenant, I don't have this problem. It loads fine._
+
```

Comment:
If there is an existing catalog and you try to upload a new one, Kill Bill assumes that you are uploading a _newer version_ of the same catalog and requires the `<catalogName>` field in the new catalog to have the same value as the existing catalog. When you see the `"Error while communicating with the Kill Bill server: Invalid catalog for tenant: [TENANT #]."`, it probably means you are trying to upload a catalog which has a different value for `<catalogName>` . To work around this, you should either delete the existing catalog before uploading a new one or ensure that the new catalog has the same `<catalogName>` as specified in the existing catalog.

Reply:
@reshmabidikar - In this scenario, does Kill Bill care if the .xml filename is different as long as `<catalogName>` is the same?

Reply:
In the original PDF, I noticed the catalog filename was different across the steps (screenshots were taken with different files as I was figuring out how things worked). Now all the displayed filenames in the screenshots are the same. Page 8 - 10. Also added notes to the beginning and end for some minor troubleshooting.

Reply:
> does Kill Bill care if the .xml filename is different as long as <catalogName> is the same?
No, Kill Bill only uses `<catalogName>`. So, if the `<catalogName>` is the same, Kill Bill treats them as the same catalog

Reply:
@reshmabidikar - I did add that <catalogName> info at the beginning of this section. At the end, I added a note about troubleshooting by deleting the old catalog.